extensions-contrib/graphite-emitter/src/main/java/io/druid/emitter/graphite/GraphiteEmitter.java

```diff
@@ -147,7 +146,7 @@ public void run()
         }
         while (eventsQueue.size() > 0 && !exec.isShutdown()) {
           try {
-            final GraphiteEvent graphiteEvent = eventsQueue.poll(DEFAULT_PUT_GET_TIMEOUT, TimeUnit.MILLISECONDS);
+            final GraphiteEvent graphiteEvent = eventsQueue.poll(graphiteEmitterConfig.getEmitWaitTime(), TimeUnit.MILLISECONDS);
```

Comment:
this probably could be a different config than the `offer` timeout. I guess the reason this is here is so you can periodically check for the emitter being stopped. I could see people wanting offer to have a 0 timeout and poll to have some nonzero timeout.